_posts/2020-07-31-kubeflow-1.1-blog-post.md

```diff
@@ -0,0 +1,39 @@
+**Kubeflow 1.1 improves ML Workflow Productivity, Isolation & Security, and GitOps**
+
+The Kubeflow Community's delivery of Kubeflow 1.1 offers users valuable ML workflow automation with Fairing and Kale along with MXNet and XGBoost distributed training operators. It extends isolation and security through the delivery of multi-user pipelines, CVE scanning, and support for Google's Private GKE and Anthos. 1.1 provides a foundation for consistent and repeatable installation and operations using GitOps methodologies powered by blueprints and kpt primitives.
```

Comment:
Could you add links to these operators' repos?

Reply:
This 1st paragraph is a summary.   I have not put links in it.